_auto/deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+### Update latest information
+# See https://github.com/endoflife-date/endoflife.date/pull/2081
+git submodule update --remote
+pip install -r requirements.txt
+# If the latest.py script fails,
+# We don't want to raise any errors
+# just undo the changes, and carry on
+if ! python _auto/latest.py ; then
```

Comment:
On my local machine the `python` command does not exist (it's python3 on Debian 11). Don't know if it will be an issue on Netlify servers.

Reply:
Tried the netlify image (`netlify/build:focal`) locally, and it didn't have a python symlink. fixed.